Lombiq.Tests.UI/SecurityScanning/SecurityScanConfiguration.cs

```diff
@@ -161,7 +161,34 @@ public SecurityScanConfiguration DisablePassiveScanRule(int id, string name = ""
     /// </param>
     public SecurityScanConfiguration DisableScanRuleForUrlWithRegex(string urlRegex, int ruleId, string ruleName = "")
     {
-        DisabledRulesForUrls[urlRegex] = new ScanRule(ruleId, ruleName);
+        _disabledRulesForUrls.Add((urlRegex, ruleId, ruleName));
+        return this;
+    }
+
+    /// <summary>
+    /// Marks a rule (can be any rule, including e.g. both active or passive scan rules) as false positive for just URLs
+    /// matching the given regular expression pattern.
+    /// </summary>
+    /// <param name="urlRegex">
+    /// The regex pattern to match URLs against. It will be matched against the whole absolute URL, e.g., ".*blog.*"
+    /// will match https://example.com/blog, https://example.com/blog/my-post, etc.
+    /// </param>
+    /// <param name="ruleId">The ID of the rule. In the scan report, this is usually displayed as "Plugin Id".</param>
+    /// <param name="justification">
+    /// A human-readable explanation of why the alert is false positive.
+    /// </param>
+    /// <remarks><para>
+    /// Marking a rule as false positive helps the development of ZAP by collecting which rules have the highest false
+    /// positive rate (see <a href="https://www.zaproxy.org/faq/how-do-i-handle-a-false-positive/">the FAQ</a>).
+    /// </para></remarks>
+    public SecurityScanConfiguration MarkScanRuleAsFalsePositiveForUrlWithRegex(string urlRegex, int ruleId, string justification)
```

Comment:
Add an optional `name` parameter like for `ConfigureActiveScanRule()` for readability, and use it where this method is used.

Reply:
Already exists, just called `justification` to emphasize that a "why" is needed.